dev/NavigationView/NavigationView.cpp

```diff
@@ -1491,7 +1492,7 @@ void NavigationView::UpdatePaneLayout()
                 }();
                 auto availableHeight = paneContentRow.ActualHeight() - itemsContainerMargin;
 
-                // The -21 below is to account for the separator height that we need to subtract.
+                // The -9px from c_paneItemsSeparatorHeight is to account for the separator height that we need to subtract.
```

Comment:
Why is it 9px? :)

Reply:
Oops, `c_paneItemsSeparatorHeight` should've been updated to 9px.